app/mailers/approval_notifier_mailer.rb

```diff
@@ -23,18 +23,37 @@ def reset_password_approval_email(user_id:)
 
   private
 
+  # permissions_users are admins as per the old permissions system
+  # accesses_users are admins as per the new permissions system
+  # we need to send emails to the superset of the two till we flip everyone over to new permissions
+
   def supervisor_emails
-    users = UserPermission.where(permission_slug: :approve_health_workers, resource: user.facility_group).map(&:user)
+    permissions_users = UserPermission.where(permission_slug: :approve_health_workers, resource: user.facility_group).map(&:user)
+    accesses_users = User.admins.where(access_level: :manager)
+      .select { |admin| admin.accessible_facilities(:manage).include?(user.facility) }
+      .reject { |admin| admin.accesses.map(&:resource_type).include?("Organization") }
```

Comment:
This could incorrectly reject people who manage some other Org, right? This is an edge case that doesn't exist today, but wanted to point it out.

Reply:
Good catch, didn't consider that. Fixing.

Reply:
Shall we encode this as a test case?